src/moonbeam/moonbase-local.js

```diff
@@ -293,12 +294,22 @@ const main = async () => {
         feePerSecond,
         keyPair: aliceKeyPair,
     });
-    const taskId = await turingHelper.api.rpc.automationTime.generateTaskId(turingAddress, providedId);
+    const taskId = await turingHelper.api.rpc.automationTime.generateTaskId(proxyOnTuring, providedId);
+
+    // Check that the task has been successfully added to the task list
+    console.log('\n5. Check that the task has been successfully added to the task list ...');
+    if (schedule.Fixed?.executionTimes[0] !== 0) {
+        console.log('\nWait for 1 minute for the execution of the XCM message to schedule task on Turing ...');
+        await delay(60000);
+        const task = await turingHelper.getAccountTask(proxyOnTuring, taskId);
+        console.log('The task has been successfully added to the task list, task: ', task.toHuman());
+    } else {
+        console.log('Immediately execution, Skip checking.');
+    }
 
     // Listen XCM events on Moonbase side
-    const additionalWaitingTime = 5 * 60 * 1000;
-    console.log(`\n5. Keep Listening XCM events on ${parachainName} until ${moment(timestampNextHour).format('YYYY-MM-DD HH:mm:ss')}(${timestampNextHour}) to verify that the task(taskId: ${taskId}, providerId: ${providedId}) will be successfully executed ...`);
-    const timeout = timestampNextHour - moment().valueOf() + additionalWaitingTime;
+    console.log(`\n6. Keep Listening XCM events on ${parachainName} until ${moment(timestampNextHour * 1000).format('YYYY-MM-DD HH:mm:ss')}(${timestampNextHour}) to verify that the task(taskId: ${taskId}, providerId: ${providedId}) will be successfully executed ...`);
```

Comment:
Why do you need the `(${timestampNextHour})` in the console log? or at least, you need to specifiy it’s a timestamp,

`(UNIX timestamp: ${timestampNextHour})`